### PR TITLE
feat: have importable home template from x-archetype-utils

### PR DESCRIPTION
**Pull request by CachedaCodes**
[EX-7633](https://searchbroker.atlassian.net/browse/EX-7633)

## Motivation and context
We wanted to have a home template to embed in the demo envs of the archetype and the clients.
The designs for the home template were replicated with the SVGs directly in the page for simplicity when importing it. I decided to get rid of the GIF for now because it was a 9 MB file that would be a pain to import and didn't seem worth it to add at this stage.

- [ ] Dependencies. If any, specify:
- [X] Open issue. If applicable, link:

## Type of change

- [ ] Bug fix (non-breaking change that fixes an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that causes existing functionality to not work as expected)
- [ ] Change requires a documentation update

## What is the destination branch of this PR?

- [X] `Main`
- [ ] Other. Specify:

## How has this been tested?

Wrap the HTML in `home-template.html` in `<html><body></body></html>` to see how it would look when embedded in a `body` and test the dependency locally to check that nothing was broken with the change to Rollup.

[EX-7633]: https://searchbroker.atlassian.net/browse/EX-7633?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: packages/x-archetype-utils/src/home/home-template.html [hunk 1]
```diff
@@ -0,0 +1,1425 @@
...
+          . Just load the Archetype JS file and initialise it!
+        </p>
+      </div>
+    </div>
+  </section>
+  <section class="x-archetype-section3 x-archetype-row">
+    <div class="x-archetype-section3__container">
+      <div class="x-archetype-section3__list x-archetype-list x-archetype-gap--06">
+        <div class="x-archetype-list x-archetype-align-center x-archetype-gap--05">
+          <svg
```
Comment (herrardo) [inline packages/x-archetype-utils/src/home/home-template.html:611]: This SVG with the circle should be paired with` Built on open-source technology` element, it was wrong in design.

image

File: packages/x-archetype-utils/src/home/home-template.html [hunk 2]
```diff
@@ -0,0 +1,1425 @@
...
+        <p class="x-archetype-section2__subtitle">
+          Please have a look to the
+          <a
+            class="x-archetype-section2__link"
+            target="_blank"
+            href="https://docs.empathy.co/develop-empathy-platform/build-search-ui/web-archetype-integration-guide.html"
+          >
+            integration documentation
+          </a>
+          . Just load the Archetype JS file and initialise it!
```
Comment (herrardo) [inline packages/x-archetype-utils/src/home/home-template.html:601]:
```suggestion
         . Just load the Archetype JS file and initialise it!
```

File: packages/x-archetype-utils/src/home/home-template.html [hunk 3]
```diff
@@ -0,0 +1,1425 @@
...
+              stroke="#243D48"
+              stroke-linecap="round"
+              stroke-linejoin="round"
+            />
+            <ellipse cx="104.303" cy="19.0451" rx="9.09091" ry="9.09091" fill="#243D48" />
+            <circle cx="18.8485" cy="19.0451" r="9.09091" fill="#243D48" />
+            <circle cx="104.303" cy="104.5" r="9.09091" fill="#243D48" />
+            <ellipse cx="18.8485" cy="104.5" rx="9.09091" ry="9.09091" fill="#243D48" />
+          </svg>
+          <p class="x-archetype-section3__title">Built on open-source technology</p>
```
Comment (herrardo) [inline packages/x-archetype-utils/src/home/home-template.html:656]:
```suggestion
          <p class="x-archetype-section3__title">
            <a
              href="https://github.com/empathyco/x"
              target="_blank"
            >
              Built on open-source technology
            </a>
          </p>
```

File: packages/x-archetype-utils/src/home/home-template.html [hunk 4]
```diff
@@ -0,0 +1,1425 @@
...
+          >
+            <circle cx="60" cy="60.8633" r="60" fill="#EDBF3B" />
+            <path
+              fill-rule="evenodd"
+              clip-rule="evenodd"
+              d="M76.2501 118.637C101.492 111.551 120 88.3688 120 60.8633C120 27.7262 93.1371 0.863281 60 0.863281C26.8629 0.863281 0 27.7262 0 60.8633C0 88.3688 18.5082 111.551 43.7499 118.637C44.9854 114.188 55.8599 74.8835 54.5455 74.3514C49.2149 72.1937 45.4545 66.9676 45.4545 60.8633C45.4545 52.8301 51.9668 46.3178 60 46.3178C68.0332 46.3178 74.5454 52.8301 74.5454 60.8633C74.5454 66.9676 70.7851 72.1937 65.4545 74.3514C64.1401 74.8835 75.0146 114.188 76.2501 118.637Z"
+              fill="#243D48"
+            />
+          </svg>
+          <p class="x-archetype-section3__title">Interoperable: Works with any API!</p>
```
Comment (herrardo) [inline packages/x-archetype-utils/src/home/home-template.html:625]:
```suggestion
          <p class="x-archetype-section3__title">
            <a
              href="https://docs.empathy.co/video-library.html?video=zNq3n_3HA2I"
              target="_blank"
            >
              Interoperable: Works with any API!
            </a>
          </p>
```

**Review by herrardo — CHANGES_REQUESTED** [review]

**Review by herrardo — APPROVED** [review]

File: packages/x-archetype-utils/package.json [hunk 5]
```diff
@@ -44,6 +44,8 @@
     "@vue/test-utils": "~1.0.3",
     "concurrently": "~7.6.0",
```
Comment (LuisMartinez15) [inline packages/x-archetype-utils/package.json:45]: **nitpick:** remove unused `concurrently` package

```suggestion
```

File: packages/x-archetype-utils/rollup.config.js [hunk 6]
```diff
@@ -0,0 +1,28 @@
+import del from 'rollup-plugin-delete';
+import typescript from 'rollup-plugin-typescript2';
+import copy from 'rollup-plugin-copy';
```
Comment (LuisMartinez15) [inline packages/x-archetype-utils/rollup.config.js:3]: **nitpick**: order `imports` alphabetically

```suggestion
import copy from 'rollup-plugin-copy';
import del from 'rollup-plugin-delete';
import typescript from 'rollup-plugin-typescript2';
```

**Review by LuisMartinez15 — DISMISSED** [review]
**praise:** good job! 👏🏿

File: packages/x-archetype-utils/src/home/home-template.html [hunk 7]
```diff
@@ -0,0 +1,1462 @@
...
+    position: relative;
+  }
+
+  .x-archetype-section1,
+  .x-archetype-section2,
+  .x-archetype-section3 {
+    padding: 0 20px;
+  }
+
+  .x-archetype-header,
```
Comment (tiborux) [inline packages/x-archetype-utils/src/home/home-template.html:1019]:
```suggestion
```

**Review by LuisMartinez15 — APPROVED** [review]

File: packages/x-archetype-utils/package.json [hunk 8]
```diff
@@ -42,8 +42,9 @@
     "@types/jest": "~27.0.3",
     "@types/node": "~16.4.12",
     "@vue/test-utils": "~1.0.3",
-    "concurrently": "~7.6.0",
     "jest": "~27.3.1",
+    "rollup": "~2.79.1",
+    "rollup-plugin-copy": "~3.4.0",
```
Comment (tiborux) [inline packages/x-archetype-utils/package.json:47]: add these dependencies:

```suggestion
    "rollup-plugin-copy": "~3.4.0",
    "rollup-plugin-delete": "~2.0.0",
    "rollup-plugin-typescript2": "~0.34.1",
```

**Review by LuisMartinez15 — CHANGES_REQUESTED** [review]
see beltran's comment regarding missing dependencies

**Review by herrardo — DISMISSED** [review]

**Comment by tiborux** [comment]
in mobile:

there is no space above text:

image

to much space between the line and the searchbox:

image

**Review by LuisMartinez15 — DISMISSED** [review]

**Review by LuisMartinez15 — APPROVED** [review]

**Pull request merged**